packages/react-core/src/next/components/Select/SelectOption.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import { MenuItemProps, MenuItem } from '../../../components/Menu';
+
+export interface SelectOptionProps extends Omit<MenuItemProps, 'ref'> {
+  /** Anything which can be rendered in a select option */
+  children?: React.ReactNode;
+  /** Classes applied to root element of select option */
+  className?: string;
+  /** Identifies the component in the Menu onSelect or onActionClick callback */
+  itemId?: any;
+  /** Flag indicating the option has a checkbox */
+  hasCheck?: boolean;
```

Comment:
Nitpick which would really depend if we'd want to make the change across all other components at some point, but this prop *could* alternatively be `hasCheckbox`.

Reply:
Hmm yeah I can see that making more sense. Is it worth switching to `hasCheckbox` here right now or is it better to remain consistent for now and update the props in a breaking change release?

Reply:
Good point. I'd be fine with keeping the prop as it is for that consistency  and doing a mass update in a breaking change release.

Reply:
Opened #8126 to track that potential breaking change